packages/cli/src/ui/AppContainer.tsx

```
@@ -1057,6 +1060,50 @@ Logging in with Google... Please restart Gemini CLI to continue.
     stdout,
   ]);
 
+  useEffect(() => {
+    const handleUserFeedback = (payload: UserFeedbackPayload) => {
+      let type: MessageType;
+      switch (payload.severity) {
+        case 'error':
+          type = MessageType.ERROR;
+          break;
+        case 'warning':
+          type = MessageType.WARNING;
+          break;
+        case 'info':
+        default:
```

nit: should we throw if an unexpected sev is encountered rather than quietly calling it info?

---

Done